src/actions/account.js

```diff
@@ -161,6 +162,14 @@ export const signInWithLedgerAddAndSaveAccounts = (accountIds) => async (dispatc
 
 const defaultCodesFor = (prefix, data) => ({ successCode: `${prefix}.success`, errorCode: `${prefix}.error`, prefix, data})
 
+const twoFactorMethod = async (method, wallet, args) => {
```

Comment:
nit: passing method name as a string breaks code navigation, so better to avoid. Like if I wanted to generalize smth similar I'd do:

```js
withAccount(account => account.sendCode())
```

```js
async function withAccount(fn) {
   const account = await wallet.getAccount(wallet.accountId)
   await fn(account)
}
```